listenbrainz_spark/ftp/download.py

```diff
@@ -119,15 +138,9 @@ def download_artist_relation(self, directory, artist_relation_dump_id=None):
             Returns:
                 dest_path (str): Local path where artist relation has been downloaded.
         """
-        self.connection.cwd('/pub/musicbrainz/listenbrainz/labs/artist-credit-artist-credit-relations/')
-        artist_relation_dump = self.list_dir()
-        req_artist_relation_dump = self.get_req_dump(artist_relation_dump, artist_relation_dump_id, 5)
-
-        self.connection.cwd(req_artist_relation_dump)
-        artist_relation_file_name = self.get_file_name(req_artist_relation_dump)
-
-        t0 = time()
-        current_app.logger.info('Downloading {} from FTP...'.format(artist_relation_file_name))
-        dest_path = self.download_dump(artist_relation_file_name, directory)
-        current_app.logger.info('Done. Total time: {:.2f} sec'.format(time() - t0))
+        ARTIST_RELATION_DUMP_ID_POS = 5
```

Comment:
Let's define this and other such constants at the top of the file they are used in and import them in the tests

Reply:
Awesome <3